test/features/resource_validation.feature

```diff
@@ -0,0 +1,136 @@
+Feature: Resource validation
+
+    Scenario: As an editor, I can create and update a resource with an uploaded validation schema
```

Comment:
can you add scenario outlines/ examples table to group some tests together
https://behave.readthedocs.io/en/latest/tutorial.html#scenario-outlines

Reply:
It's possible, but there's too much different behaviour between the different scenarios for it to really be worthwhile, I think. I could combine the validation schema and validation options scenarios into one, if you want, but it still wouldn't be an outline.